doc/man3/flux_kvs_lookup.adoc

```diff
@@ -151,6 +152,11 @@ changed.  Unlike FLUX_KVS_WATCH_UNIQ, the key being watched need not
 be mentioned in a transaction.  This may occur under several
 scenarios, such as a parent directory being altered.
 
+FLUX_KVS_WAITCREATE::
+If a KVS key does not exist, wait for it to exist before returning.
+This flag may be specified along with FLUX_KVS_WATCH.  The lookup can
```

Comment:
Should this say "with or without FLUX_KVS_WATCH"?

Reply:
I see, that would probably be clearer.